confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConstant.java

```diff
@@ -42,6 +42,7 @@ public class ConfigNodeConstant {
   public static final String DATA_DIR = "data";
   public static final String CONF_DIR = "conf";
   public static final String CONSENSUS_FOLDER = "consensus";
+  public static final String PROCEDURE_FOLDER = "proc";
```

Comment:
How about storing this under the system folder, reducing a parameter for users

Besides, we could rename this to procedure, proc looks like process...